doc.config.json

```diff
@@ -23,6 +23,10 @@
                 "title": "Getting Started",
                 "path": "/docs/getting-started.mdx"
               },
+              {
+                "title": "Deploying",
+                "path": "/docs/deploy-your-app.mdx"
+              },
```

Comment:
Dfeels a bit odd to have `Deploying` and `Deployment` as two pages in the same menu. Is there a way we could merge this content?

Reply:
Agreed, I don;t think merging it is the right solution.
We want the deploy steps to be short and easily reachable - I've moved it into the guides section and renamed it to Deploying with the CLI